resources/email_templates/once_the_speaker_confirms_for_the_talk.md

```diff
@@ -0,0 +1,24 @@
+Hello,
+
+Thank you for confirming the workshop/talk/hands-on. Your session will truly benefit the group.
+
+Now to plan the session can you please provide the following information, whichever you are okay in sharing in with public.
+
+1. Your full name or the name you prefer to be called by the community.
+2. The pronoun you are comfortable with.
+3. Your introduction, a short bio.
+4. Title of the workshop/talk/hands-on
+5. Short description of the workshop/talk/hands-on
+6. The duration you are looking for the talk.
+7. The name of your organization and if you are comfortable sharing them.
+8. Are you okay if I tag you on Social media? Please share your social media handles (Mastodon, Linkedin, and Twitter) if you are comfortable.
+9. Are you there in the Ansible Matrix? If not, then please follow. Can we share your Matrix username and social media handles in the meetup event page? It will be helpful for the attendees and broader community members to communicate with you.
+
+Let us know what information you are okay to share. Also please specify the adminstrative stuff, requirements for your presentation or any other details you want us know or take care.
+
+Please share the details above by {date}. We intend to announce the event on {date}.
```

Comment:
```suggestion
To ensure your participation, please provide your session details by {deadline date}. We plan to announce the event on {announce date} and need to confirm all sessions before the announcement.
```

Reply:
I would move this paragraph to the top of the section so the time sensitivity is more prominent to the reader.